prime-router/docs/api/delivery.yml

```diff
@@ -8,6 +8,39 @@ info:
     email: reportstream@cdc.gov
   version: 0.2.0-oas3
 paths:
+  /v1/receivers/{receiverFullName}/deliveries:
+    post:
+      description: Returns a list of reports that have been delivered to the receiver
+      security:
+        - OAuth2: [ user ]
+        - ApiKeyAuth: [ ]
```

Comment:
securitySchemes is missing

```
  securitySchemes:
    OAuth2:
      $ref: 'https://raw.githubusercontent.com/CDCgov/prime-reportstream/master/prime-router/docs/api/components/security_schemes.yml#/OAuth2'
```

Reply:
We've been using `security` and `securitySchemes` somewhat interchangeably on our API docs. I think it's worth discussing the structure of those components. 
I noticed there's different use cases for each: https://swagger.io/docs/specification/authentication/